mpd/mpd.go

```diff
@@ -57,8 +57,11 @@ type MPD struct {
 }
 
 type Period struct {
-	AdaptationSets []*AdaptationSet `xml:"AdaptationSet,omitempty"`
-	BaseURL        string           `xml:"BaseURL,omitempty"`
+	BaseURL         string           `xml:"BaseURL,omitempty"`
```

Comment:
BaseURL was a string before the changes so left it as is to avoid breaking anything referencing it.